egs/sprakbanken/s5/local/chain/run_tdnn.sh

```diff
@@ -0,0 +1 @@
+tuning/run_tdnn_1b.sh
```

Comment:
Actually no need to have the 'just-lstm' number, generally TDNN+LSTM will work better.